README.md

```diff
@@ -1,13 +1,32 @@
 # Project Todos
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Building a app with todo's in Js with React/Redux
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I started this project with getting the a basic tasklist and input as well as setting up a store with the most essential reducers and states. First focus was just that fundamental fuction of the user being able to add and then tick tasks when done. 
```

Comment:
OMG! 🐱‍🏍
I'm super impressed and I don't think I'm able to find any real room-for-improvements.
Can't say that I understand everything , but that's not because of your code but because I'm on a completely different level 🙃
The only thing might be that there are so many functions and options that it was easy (for ME) to get lost in them, other than that I think it's a killer project 💪🏼